Policies/github-acceptable-use-policies.md

```diff
@@ -50,10 +50,21 @@ While using the Service, under no circumstances will you:
 
 - violate the privacy of any third party, such as by posting another person's personal information without consent.
```

Comment:
This is also a gray area. What is considered personal information - is a single mailbox or name considered personal information?